### DISCUSSION TITLE
Adding placeholders in head

### DISCUSSION THREAD

**Discussion by simpel** [Sitecore/jss#1795] [Help]
### Description

Hi!

We're developing a JSS headless NextJS site and trying to place a <Placeholder> in the <Head> tag in the Layout.tsx file but it does not seem to work, why? And how should we solve it? We're doing this so we can pass Data to a <MetaData> componment that should render a bunch of stuff

It complains, but if I move the Placeholder outside of <Head> it works but...yea, the stuff wont be in the head... 
![image](https://github.com/Sitecore/jss/assets/232024/6811b55f-d1be-42d3-a66a-a95538ac324d)

**Comment by yavorsk** [comment]
Hi @simpel :)
So yes, placing placeholder inside the <Head> won't work. Placeholders are intended to be used in visual editing areas. You have options to get around this limitation though:
- you can add route level fields to render data in <Head>;
- you can leverage nextjs' Head component https://nextjs.org/docs/pages/api-reference/components/head : 1/ create your 'Metadata' component which uses Nextjs' <Head> to render things in the <Head> element; 2/ place that component in any other placeholder inside the body;
